extension/goext/environment.go

```diff
@@ -22,6 +22,7 @@ type IEnvironment interface {
 	Config() IConfig
 	Core() ICore
 	Logger() ILogger
+	LoggerForSchema(schema ISchema) ILogger
```

Comment:
I agree with @morrisson , it's not very clean approach.

Reply:
Let's follow this discussion in Jira